files/default/sql/ddl/updates/3.4.0.sql

```diff
@@ -158,6 +158,18 @@ ALTER TABLE `hopsworks`.`feature_group` ADD COLUMN `topic_name` VARCHAR(255) DEF
 ALTER TABLE `hopsworks`.`project` ADD COLUMN `topic_name` VARCHAR(255) DEFAULT NULL;
 
 SET SQL_SAFE_UPDATES = 0;
+-- deletes unused topics (fg was deleted)
```

Comment:
why do we endup with alot of orphaned topics in the first place?

Reply:
It happens because we dont delete topics from online enabled feature groups (with suffix _onlinefs). We keep kafka topics in order to avoid consumers (onlinefs) getting blocked. This is something we should fix, but right now there is no solution.